### ISSUE TITLE
Paste images to the chatbox

### ISSUE THREAD

**Issue by juansgv** [logancyang/obsidian-copilot#575]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by logancyang** [comment]
This is in my plan next.

**Comment by tobalsan** [comment]
hey @logancyang 
Thanks for the awesome plugin.

Just to add my vote to this, I deal with Excalidraw flowcharts that could be converted into images to be interpreted by Vision-capable LLMS, so this feature would be a fantastic addition.

**Comment by logancyang** [comment]
@tobalsan definitely. Multimodal agentic RAG will be gradually introduced in the next few releases. I will make an announcement soon. Stay tuned!
